api-test/src/test/java/org/openmrs/module/metadatasharing/PreserveConceptIdsTest.java

```diff
@@ -75,13 +77,17 @@ public void shouldNotPreserveConceptIdAcrossServersIfImportingServerDoesntReques
 		runShareTest(new ShareTestHelper() {
 			
 			Integer newConceptId = null;
+
+			Concept concept;
 			
 			@Override
 			public List<?> prepareExportServer() throws Exception {
 				
 				Concept c = new Concept();
 				c.addName(new ConceptName("Test", Locale.US));
-				
+				c.addDescription(new ConceptDescription("Description", Locale.ENGLISH));
+				concept = c;
```

Comment:
Do we need the above line?

Reply:
In platform 2.0, the concept must have a description otherwise it gives the following error when saving.

`'Concept #null' failed to validate with reason: descriptions: Concept.description.atLeastOneRequired`